qemu/tests/virtio_fs_share_data.py

```diff
@@ -210,6 +211,15 @@ def get_viofs_exe(session):
                                        io_timeout).stdout_text.strip().split()[0]
                 if md5_guest != md5_host:
                     test.fail('The md5 value of host is not same to guest.')
+                if reboot_flag == "yes":
+                    vm.reboot(session)
+                    vm.verify_alive()
+                    session = vm.wait_for_login()
+                    logging.info("Check virtiofsd on host.")
+                    cmd_ps_virtiofsd = "ps -ef |grep virtiofsd |grep -v grep"
+                    status, output = session.cmd_status_output(cmd_ps_virtiofsd)
+                    if not status and len(output.splitlines) != 2:
```

Comment:
`len(output.splitlines) != 2:`, I think this way to judgement is not accurate, since there is such a scenario that there are more than one daemons are running, for example zombie process.

Reply:
You mean maybe there are more than 2 daemons are running. I can only see two process when booting up one vm.

Reply:
Yes, if the previous case failed and the virtiofs daemon is not quit, then we run this case, `len(output.splitlines)` will get more than two lines.